docs/streams/developer-guide.html

```diff
@@ -893,13 +893,15 @@
             KeyValue&lt;String, Long&gt; next = range.next();
             System.out.println("count for " + next.key + ": " + value);
           }
+          range.close();
```

Comment:
Could we also add a comment above saying that it is the caller's responsibility to close the iterator; otherwise it may lead to OOM and leaked file handlers depending on the state store implementation.

Ditto below.